### ISSUE TITLE
[enhancement] Option to set the maximum number of the badge count

### ISSUE THREAD

**Issue by applepipe** [frappe/erpnext#8534]
We should be able to choose what is the maximum count number on badges

**Comment by adityaduggal** [comment]
I suppose you are talking about the 20+ number that comes in front of the transaction on the awesome bar

**Comment by applepipe** [comment]
Yes, also on icons, links to doctypes, etc. Every badge, some time ago I started this topic https://discuss.erpnext.com/t/icon-badge-count/13834 but it would be nice to have a configurable option
